Sources/PackageDescription/Version.swift

```diff
@@ -25,13 +25,17 @@ public struct Version {
     }
 
     public init?(_ characters: String.CharacterView) {
-        let components = characters.split(".", maxSplit: 2, allowEmptySlices: true).map(String.init).flatMap{ Int($0) }.filter{ $0 >= 0 }
+        let components = Version.componentsFromCharacters(characters)
         guard components.count == 3 else { return nil }
 
         self.major = components[0]
         self.minor = components[1]
         self.patch = components[2]
     }
+    
+    private static func componentsFromCharacters(characters: String.CharacterView) -> [Int] {
```

Comment:
Since this is a function used by two separate structs in the same file, I think it would be better as just a top level private function rather than a static function on the `Version` struct.